convert/testdata/granite-3.0-2b-base.json

```diff
@@ -0,0 +1,396 @@
+{
+  "general.architecture": "granite",
+  "general.file_type": "1",
+  "general.parameter_count": "2533525504",
+  "general.quantization_version": "2",
+  "granite.attention.head_count": "32",
+  "granite.attention.head_count_kv": "8",
+  "granite.attention.layer_norm_rms_epsilon": "1e-05",
+  "granite.attention.scale": "0.015625",
+  "granite.block_count": "40",
+  "granite.context_length": "4096",
+  "granite.embedding_length": "2048",
+  "granite.embedding_scale": "12",
+  "granite.feed_forward_length": "8192",
+  "granite.logit_scale": "8",
+  "granite.residual_scale": "0.22",
```

Comment:
Note that granite is super close to llama - the main difference is that the head dim isn't used, and that the following multipliers are converted correctly.
```
granite.residual_scale
granite.embedding_scale
granite.logit_scale
granite.attention.scale
```

Reference from Llama Cpp's conversion [script](https://github.com/ggerganov/llama.cpp/blob/master/convert_hf_to_gguf.py#L4616), where the granite model conversion is a subclass of llama with some extra handling for these